<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterForm</class>
 <widget class="QWidget" name="ParameterForm">
  <layout class="QVBoxLayout" name="outerLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <!-- Mode -->
     <item row="0" column="0"><widget class="QLabel" name="modeLbl"><property name="text"><string>Mode</string></property></widget></item>
     <item row="0" column="1">
      <widget class="QComboBox" name="modeCombo">
       <item><property name="text"><string>AOO</string></property></item>
       <item><property name="text"><string>VOO</string></property></item>
       <item><property name="text"><string>AAI</string></property></item>
       <item><property name="text"><string>VVI</string></property></item>
      </widget>
     </item>

     <!-- Rate limits -->
     <item row="1" column="0"><widget class="QLabel" name="lrlLbl"><property name="text"><string>LRL (bpm)</string></property></widget></item>
     <item row="1" column="1"><widget class="QSpinBox" name="lrlSpin"><property name="minimum"><number>30</number></property><property name="maximum"><number>175</number></property><property name="value"><number>60</number></property></widget></item>

     <item row="2" column="0"><widget class="QLabel" name="urlLbl"><property name="text"><string>URL (bpm)</string></property></widget></item>
     <item row="2" column="1"><widget class="QSpinBox" name="urlSpin"><property name="minimum"><number>50</number></property><property name="maximum"><number>200</number></property><property name="value"><number>120</number></property></widget></item>

     <!-- Atrial -->
     <item row="3" column="0"><widget class="QLabel" name="aAmpLbl"><property name="text"><string>Atrial Amplitude (V)</string></property></widget></item>
     <item row="3" column="1"><widget class="QDoubleSpinBox" name="aAmpSpin"><property name="minimum"><double>0.0</double></property><property name="maximum"><double>7.0</double></property><property name="singleStep"><double>0.1</double></property><property name="value"><double>3.5</double></property></widget></item>

     <item row="4" column="0"><widget class="QLabel" name="aPwLbl"><property name="text"><string>Atrial Pulse Width (ms)</string></property></widget></item>
     <item row="4" column="1"><widget class="QDoubleSpinBox" name="aPwSpin"><property name="minimum"><double>0.05</double></property><property name="maximum"><double>1.90</double></property><property name="singleStep"><double>0.05</double></property><property name="value"><double>0.4</double></property></widget></item>

     <!-- Ventricular -->
     <item row="5" column="0"><widget class="QLabel" name="vAmpLbl"><property name="text"><string>Ventricular Amplitude (V)</string></property></widget></item>
     <item row="5" column="1"><widget class="QDoubleSpinBox" name="vAmpSpin"><property name="minimum"><double>0.0</double></property><property name="maximum"><double>7.0</double></property><property name="singleStep"><double>0.1</double></property><property name="value"><double>3.5</double></property></widget></item>

     <item row="6" column="0"><widget class="QLabel" name="vPwLbl"><property name="text"><string>Ventricular Pulse Width (ms)</string></property></widget></item>
     <item row="6" column="1"><widget class="QDoubleSpinBox" name="vPwSpin"><property name="minimum"><double>0.05</double></property><property name="maximum"><double>1.90</double></property><property name="singleStep"><double>0.05</double></property><property name="value"><double>0.4</double></property></widget></item>

     <!-- Refractory -->
     <item row="7" column="0"><widget class="QLabel" name="arpLbl"><property name="text"><string>ARP (ms)</string></property></widget></item>
     <item row="7" column="1"><widget class="QSpinBox" name="arpSpin"><property name="minimum"><number>150</number></property><property name="maximum"><number>500</number></property><property name="value"><number>250</number></property></widget></item>

     <item row="8" column="0"><widget class="QLabel" name="vrpLbl"><property name="text"><string>VRP (ms)</string></property></widget></item>
     <item row="8" column="1"><widget class="QSpinBox" name="vrpSpin"><property name="minimum"><number>150</number></property><property name="maximum"><number>500</number></property><property name="value"><number>250</number></property></widget></item>
    </layout>
   </item>

   <!-- Buttons row -->
   <item>
    <layout class="QHBoxLayout" name="buttonRow">
     <item><spacer name="btnLeft"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeHint" stdset="0"><size><width>40</width><height>20</height></size></property></spacer></item>
     <item><widget class="QPushButton" name="saveBtn"><property name="text"><string>Save</string></property></widget></item>
     <item><widget class="QPushButton" name="loadBtn"><property name="text"><string>Load</string></property></widget></item>
     <item><widget class="QPushButton" name="clearBtn"><property name="text"><string>Clear</string></property></widget></item>
     <item><spacer name="btnRight"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeHint" stdset="0"><size><width>40</width><height>20</height></size></property></spacer></item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
